hazelcast/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -1386,160 +1414,214 @@ protected void handleHelp(String command) {
+        println("  mm.get <key>                            returns the value of given key from the multimap");
+        println("  mm.remove <key>                         removes the entry of given key from the multimap");
+        println("  mm.size                                 size of the multimap");
+        println("  mm.clear                                clears the multimap");
+        println("  mm.destroy                              destroys the multimap");
+        println("  mm.iterator [remove]                    iterates the keys of the multimap, remove if specified");
+        println("  mm.keys                                 iterates the keys of the multimap");
+        println("  mm.values                               iterates the values of the multimap");
+        println("  mm.entries                              iterates the entries of the multimap");
+        println("  mm.lock <key>                           locks the key");
+        println("  mm.tryLock <key>                        tries to lock the key and returns immediately");
+        println("  mm.tryLock <key> <time>                 tries to lock the key within given seconds");
+        println("  mm.unlock <key>                         unlocks the key");
+        println("  mm.stats                                shows the local stats of the multimap");
         println("");
     }
 
     private void printExecutorServiceCommands() {
-        println("-- Executor Service commands:");
-        println("execute <echo-input>                            //executes an echo task on random member");
-        println("executeOnKey <echo-input> <key>                  //executes an echo task on the member that owns the given key");
-        println("executeOnMember <echo-input> <memberIndex>         //executes an echo task on the member with given index");
-        println("executeOnMembers <echo-input>                      //executes an echo task on all of the members");
-        println("e<threadcount>.simulateLoad <task-count> <delaySeconds>        //simulates load on executor with given number "
-                + "of thread (e1..e16)");
-
+        printlnBold("Executor Service commands:");
+        println("  execute <echo-input>                    executes an echo task on random member");
+        println("  executeOnKey <echo-input> <key>         executes an echo task on the member that");
+        println("                                            owns the given key");
+        println("  executeOnMembers <echo-input>           executes an echo task on all of the members");
+        println("  executeOnMembers <echo-input>           executes an echo task on the member with");
+        println("                     <memberIndex>          given index");
+        println("  e<threadcount>.simulateLoad             simulates load on executor with given number");
+        println("          <task-count> <delaySeconds>       of thread (e1..e16)");
```

Comment:
```suggestion
        println("  executeOnMembers <echo-input> <memberIndex>");
        println("    executes an echo task on the member with given index");
        println("");
        println("  e<threadcount>.simulateLoad  <task-count> <delaySeconds>");
        println("    simulates load on executor with given number of thread (e1..e16)");
```

maybe this format would be better for the commands that doesn't fit

Reply:
I think it is better to align descriptions, so I'll keep it as it is